website/content/docs/auth/login-mfa/index.mdx

```diff
@@ -62,8 +62,10 @@ In the Single-phase login, the required MFA information is embedded in a login r
 the `X-Vault-MFA` header. In this case, the MFA validation is done
 as a part of the login request.
 
-MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. The format of
-the header is `mfa_method_id[:passcode]`. The item in the `[]` is optional. If there are multiple MFA methods that need to be validated, a user can pass in multiple `X-Vault-MFA` HTTP headers.
+MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. Before Vault 1.13.0, the format of
+the header is `mfa_method_id[:passcode]` for TOTP, Okta, and PingID. However, for Duo, it is `mfa_method_id[:passcode=<passcode>]`.
+The item in the `[]` is optional. From Vault 1.13.0, the format is consistent for all supported MFA methods, and one can use either of the above two formats.
+If there are multiple MFA methods that need to be validated, a user can pass in multiple `X-Vault-MFA` HTTP headers.
```

Comment:
MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. Before Vault 1.13.0, the format of
the header is `mfa_method_id[:passcode]` for TOTP, Okta, and PingID. However, for Duo, it is `mfa_method_id[:passcode=<passcode>]`.

This reads as before vault 1.13.0 the duo method is different. Is that correct?

Reply:
Yes, that is correct